compiler/passes/src/flattening/flatten_statement.rs

```diff
@@ -273,7 +273,9 @@ impl StatementReconstructor for Flattener<'_> {
         let guard = self.construct_guard();
 
         match input.expression {
-            Expression::Unit(_) | Expression::Identifier(_) => self.returns.push((guard, input)),
+            Expression::Unit(_) | Expression::Identifier(_) | Expression::Access(_) => {
```

Comment:
I added it because there was a compiler panic when a return statement contained an access expression. Looks like it might not be processed in SSA to ensure proper CG.  https://github.com/AleoHQ/leo/blob/e6ba5069dc115837b1df361099db3db265f75676/compiler/passes/src/static_single_assignment/rename_statement.rs#L359